### ISSUE TITLE
grafana_auth_bypass - No Vhost Option

### ISSUE THREAD

**Issue by random-robbie** [rapid7/metasploit-framework#13296]
## Steps to reproduce

How'd you do it?

1.  use admin/http/grafana_auth_bypass
2. show options

msf5 auxiliary(admin/http/grafana_auth_bypass) > show options

Module options (auxiliary/admin/http/grafana_auth_bypass):

   Name       Current Setting  Required  Description
   ----       ---------------  --------  -----------
   COOKIE                      no        Decrypt captured cookie
   RHOSTS     xxx.xxx.xxx.xxx     yes       Address of target
   RPORT      443              yes       Port of target
   SSL        true             yes       set SSL/TLS based connection
   TARGETURI  /                no        Base URL of grafana instance
   THREADS    1                yes       The number of concurrent threads (max one per host)
   USERNAME   admin            no        Valid username
   VERSION    2-4              yes       Grafana version: "2-4" or "5" (Accepted: 2-4, 5)

## Expected behavior
Set VHOST sets the host name and allows to execute against the vhost.

## Current behavior

unable to set VHOST on current target so when sending the request via the IP it does not work.

What happens instead?
Current grafana host only works with hostname so currently getting 502 error when visiting IP only.

### Metasploit version

Framework: 5.0.86-dev
Console  : 5.0.86-dev

### I installed Metasploit with:
- [ X] Docker

### OS

What OS are you running Metasploit on?
Ubuntu 18.04

**Comment by wvu-r7** [comment]
@smcintyre-r7

**Comment by smcintyre-r7** [comment]
Yeah it's an external module so it doesn't get all the datastore options that native Metasploit modules implemented in Ruby do. I'm sure it could be done using the `requests` library though.

This sounds like a good suggestion.
